nss-copper-dragonflies/BangazonAPI
# Brj employee

# Description

Created the Employee Resource consisting of the Employee Model, Employee Controller, and Employee Test. Added functionality to GET all, GET single, POST, and PUT employees.

Fixes # 6

## Type of change



# Testing Instructions

1. `git fetch -a`
1. `git checkout brj-employee`
1. run all tests for the employee controller
1. in Postman confirm that you are able to successfully GET all, Get single, POST, and PUT
1. have a blessed day!!!!!!



# Checklist:
